data_structures/linkedlist_queue.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Please fix these comments by (1) rewording them (2) noting the proper time complexity of O(1) (3) using a JSDoc comment for all of this.

Reply:
fixed it!